### DISCUSSION TITLE
Discord channel

### DISCUSSION THREAD

**Discussion by ilkkapeltola** [thumbmarkjs/thumbmarkjs#74] [Announcements]
Join the [Thumbmark Discord Channel](https://discord.gg/PAqxQ3TnDA) if you want to collaborate with others.
I personally might be there occasionally too 😁

**Comment by MarkoXR** [comment]
Hi @ilkkapeltola,
The invite link is invalid 😕
<img width="487" alt="Screenshot 2024-12-16 at 10 53 49" src="https://github.com/user-attachments/assets/5443aad6-14ce-4af5-863c-7b5fa9cc09cc" />

**Reply by ilkkapeltola** [reply]
Could you try now? I'm not sure what happened, but the correct link seems to be https://discord.gg/UM2sawXX

**Reply by MarkoXR** [reply]
Works now! 👍

**Comment by MatthewSBarnes** [comment]
Hey folks, is this still active? @MarkoXR @ilkkapeltola ?

**Reply by MarkoXR** [reply]
@MatthewSBarnes The link at the top is working for me (not the one in the comments)

**Reply by MatthewSBarnes** [reply]
Ah oops! Thank you!

**Comment by jgabriel98** [comment]
It seems that all those links are broken.
Could anyone provide me the discord channel link?

**Reply by ilkkapeltola** [reply]
There's a link on the repo readme that is working. I'll avoid pasting it here in case the link needs to be reset for some reason, adding yet another broken link.

But a working link is on the repo.
